packages/migrate/src/commands/DbPull.ts

```diff
@@ -252,9 +252,9 @@ Some information will be lost (relations, comments, mapped fields, @ignore...),
       debug(`Introspection Schema Version: ${introspectionResult.version}`)
     } catch (e: any) {
       introspectionSpinner.failure()
-      if (e.code === 'P4001') {
+      if (e.code === 'P1003') {
         if (introspectionSchema.trim() === '') {
-          throw new Error(`\n${chalk.red.bold('P4001 ')}${chalk.red('The introspected database was empty:')} ${
+          throw new Error(`\n${chalk.red.bold('P1003 ')}${chalk.red('The introspected database was empty:')} ${
```

Comment:
I think that this shows a breaking change in our public API here before we had the error code `P4001` and now it disappears (changed to P1003)
We need to find another way and keep the old codes or go for a major version bump.

Reply:
Answered here: https://prisma-company.slack.com/archives/C04HZ2EQS65/p1673446997564369?thread_ts=1673440456.031899&cid=C04HZ2EQS65

Reply:
TODO:
We now we have 2 cases
- Database is empty
- Database does not exist
We need to check if we have tests for both in introspection

Reply:
I've added human-friendly error support for both `P4001` and `P1003`. Please check https://github.com/prisma/prisma/pull/16766/commits/9c820ae3348de82991defb068750189426bbfaa5